exercises/two-fer/README.md

```diff
@@ -0,0 +1,38 @@
+# Two Fer
```

Comment:
Out of curiosity: how did you create this file? Did you copy-paste the contents of the C# one and updated it? If so, there is actually a nicer (and recommended) way to do this, by using the [configlet tool](https://github.com/exercism/configlet). configlet is a tool built to help maintain tracks. It has got linting functionality and also a documentation generation option. With configlet the `README.md` file of an exercise can be generated. For that, it uses the exercise's information in the [problem-specifications repo](https://github.com/exercism/problem-specifications), and then uses that in a template to generate the readme: https://github.com/exercism/vbnet/blob/master/config/exercise_readme.go.tmpl

See https://github.com/exercism/csharp/blob/master/update-docs.ps1#L32 for an example on how to run it for all exercises. Using the `bin/fetch-configlet` script, you can download a local version (I've submitted a PR to update the fetch script to the latest version: https://github.com/exercism/vbnet/pull/53).

Reply:
I did actually use the confliglet script to generate this README.md